cmd/helm/plugin_remove.go

```diff
@@ -67,9 +68,12 @@ func (pcmd *pluginRemoveCmd) run() error {
 	for _, name := range pcmd.names {
 		if found := findPlugin(plugins, name); found != nil {
 			if err := removePlugin(found, pcmd.home); err != nil {
-				return err
+				fmt.Fprintf(pcmd.out, "Failed to remove plugin %s, got error (%v)\n", name, err)
+			} else {
+				fmt.Fprintf(pcmd.out, "Removed plugin: %s\n", name)
 			}
-			fmt.Fprintf(pcmd.out, "Removed plugin: %s\n", name)
+		} else {
+			fmt.Fprintf(pcmd.out, "Plugin: %s not found\n", name)
```

Comment:
Right now this will still return 0 when exiting (same with line 71). This should return `fmt.Errorf` so it errors out properly